packages/gatsby-plugin-typography/src/gatsby-node.js

```diff
@@ -22,7 +23,7 @@ const typography = new Typography()
 module.exports = typography`
   }
 
-  const dir = `${__dirname}/.cache`
+  const dir = `${program.directory}/.cache`
```

Comment:
This actually is what we want. We want to write the file into the same directory as where the gatsby-ssr.js file is so it can require it.

Reply:
I'll push a commit to revert this.

Reply:
The problem is ssr files don't know where the root of the site is at.

Reply:
Christopher was trying to help me figure out why it was failing and that line was something he thought might be causing it. Thanks for taking a look at this!